src/main/java/io/fusionauth/domain/api/identityProvider/IdentityProviderLoginRequest.java

```diff
@@ -33,6 +33,8 @@
 
   public UUID identityProviderId;
 
+  public boolean bypassAuthTokenRequest;
```

Comment:
This will work, however - this is a Twitter specific configuration and this object is generic for all IdPs. 

So if we wanted to keep it here, we should probably name it accordingly. Or, you can add it to the `data` object and then we can just document the flag that way. 

```
data.bypassAuthTokenRequest
```

Reply:
what do you think of implementing user's suggestion:

"If oauth_verifier is not given then the oauth_token and oauth_token_secret should be considered as access tokens"

If oauth_verifier is missing, we treat the other two fields as access tokens and skip twitter API call.

Reply:
I'm ok with that I think - initially I was thinking it would be better to make this an explicit request on behalf of the API caller. 

However, we do have a debug option on this IdP, so if we add debug log entries to this workflow to indicate what we are doing based upon the user input to make it clear that we are bypassing the Access Token request because the `oauth_verifier` has been omitted then it should be easy enough to debug if they did not intend to omit the `oauth_verifier`. 

Sounds good to me.